src/app/components/header.tsx

```diff
@@ -0,0 +1,60 @@
+"use client"
+import clsx from "clsx";
+import { AppContainer } from "../globale/container";
+import { FC } from "react";
+import { useTheme } from "next-themes";
+
+
+
+type HeaderProps = {
+  isDarkMode?: boolean;
+  // setIsDarkMode?:  Dispatch<SetStateAction<boolean>>;
+}
+
+const Header:FC<HeaderProps> = ({isDarkMode}) => {
+  const { theme, setTheme } = useTheme()
+  const hanldeToggleTheme = () => {
+    console.log(theme, 'new theme');
+    setTheme((prevTheme) => theme === "dark" ? "light" : "dark")
+    console.log(theme);
+    
+  }
+
+  return (
+    <header
+      className={clsx(
+        "py-3 px-3 lg:py-2",
+        "transition-all duration-300",
+        "shadow-md", "bg-primary-foreground"
+      )}
+    >
+      <AppContainer>
+        <div className="flex justify-between items-center  text-[1rem]">
+          <h3 className={clsx("font-bold lg:text-[2rem]")}>Where in the world?</h3>
+          <button
+            className="flex gap-2 items-center lg:text-[1.5rem]"
+            onClick={hanldeToggleTheme}
+          >
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="24"
+              height="24"
+              viewBox="0 0 24 24"
+              fill={isDarkMode ? 'white' : 'white'}
```

Comment:
What is the point of checking isDarkMode if the outcome is always "white" whether isDarkMode is true or false?